### DISCUSSION TITLE
AXIS engine could not find a target service

### DISCUSSION THREAD

**Discussion by owlbrudder** [eclipse-birt/birt#1007] [Q&A] [answered]
I am aware this has been asked a million times, but please forgive me as I am a total newbie to BIRT.

Just installed Eclipse on a new system then installed BIRT and everything in the View Report menu works brilliantly except for "View Report In Web Viewer" which throws an error message:
![Screenshot from 2022-07-01 10-09-42](https://user-images.githubusercontent.com/20263907/176811290-4fdf2fe0-57ae-440f-913d-1d91c134aa64.png)

From what I gather through Googling the message it seems I may be missing a configuration file somewhere, but I am not smart enough to know how to fix it. Any help would be greatly appreciated.

**Comment by wimjongman** [comment, marked as answer]
Hey Doug. We need more info.

BIRT version, Eclipse version, etc. You can attach the config data as a file (DO NOT PASTE IT). Config data can be obtained from Eclipse by using 

CRTL+3 and "Copy configuration data to clipboard" and "paste in file" and "attach file here"

**Reply by owlbrudder** [reply]
Good afternoon Mr. Jongman. That name seems familiar to me - did you help me last year when I had trouble with WindowBuilder? Sadly in December 2021 my computers were drowned by the storm surge from Super Typhoon Odette here in the Philippines and I am only now getting back to a working system.

The good news is that my new computer has enough memory and cores to handle BIRT, which the old one did not, so I am progressing.

I am now using Fedora 36 on a Ryzen 5600G CPU with 32Gb RAM.. The Java I have installed is:
`[doug@dennis ~]$ java --version
openjdk 17.0.3 2022-04-19
OpenJDK Runtime Environment 21.9 (build 17.0.3+7)
OpenJDK 64-Bit Server VM 21.9 (build 17.0.3+7, mixed mode, sharing)
[doug@dennis ~]$ 
`

Attached please find the configuration data you asked for.
[axis.txt](https://github.com/eclipse/birt/files/9026915/axis.txt)

**Reply by wimjongman** [reply]
> did you help me last year when I had trouble with WindowBuilder? 

I was designated as your personal assistant, Doug. ;)

> org.eclipse.birt (4.9.0.v202203150031) "Business Intelligence and Reporting Tools"

You seem to be running the march version of 4.9 can you first try to upgrade from the latest nightly? 

https://download.eclipse.org/birt/update-site/snapshot/

**Reply by owlbrudder** [reply]
As always, your advice is excellent sir. I installed the latest snapshot, which actually performed an update, and the Web Viewer is now working. Thank you very much indeed.

**Comment by owlbrudder** [comment]
How nice to make contact again sir. I will not install WindowBuilder on my
new machine until the current issues are solved .. one challenge at a
time.  "8-D

The computer is busy right now, but I will install the latest nightly
version as soon as possible and get back to you with the result.

**Reply by TheLaserNinja** [reply]
Curious, what problems were you running into with WindowBuilder? I have had them installed side by side in the past with no issues. In fact, it was what I was using to build the UI's for ODA's I built in BIRT.

**Reply by owlbrudder** [reply]
I would be very glad to have a solution as I prefer the Eclipse IDE, but it does not work for me.

The problem is the wireframes are offset from their controls in design view. See this discussion for details.
https://github.com/eclipse/windowbuilder/discussions/306
